src/compiler/crystal/codegen/codegen.cr

```diff
@@ -2036,7 +2036,7 @@ module Crystal
       if raise_overflow_fun = @raise_overflow_fun
         check_main_fun RAISE_OVERFLOW_NAME, raise_overflow_fun
       else
-        raise "BUG: __crystal_raise_overflow is not defined"
+        raise "ERROR: missing __crystal_raise_overflow function, either use std-lib's prelude or define it"
```

Comment:
Can we have this be a `LocationlessException`? That should suppress the "You have found a bug in the compiler" message, no?

Reply:
oh heck, i spent a few hours trying to figure out how to suppress that and never realized that's what `LocationlessException` was for lol